### DISCUSSION TITLE
Get all total items in certain column

### DISCUSSION THREAD

**Discussion by urkiddennn** [pocketbase/pocketbase#5207] [Q&A] [answered]
I want to get all the total number of a certain column in the collections

**Comment by ganigeorgiev** [comment, marked as answer]
Without details about your collections structure and your use case it is not clear what actually you are asking or trying to do.

If you want to perform an aggregation (eg. `SUM`) on a specific field you can create a helper [View collection](https://pocketbase.io/docs/collections/#view-collection) with a query like:

```sql
SELECT
    (ROW_NUMBER() OVER()) as id,
    SUM(someField) as total
FROM yourCollection
```

If you need further help please don't treat the discussions as llm chat and try to elaborate a little more on your use case.

**Reply by urkiddennn** [reply]
sorry about that, what I'm trying to ask was if there is a way for me to get all the items from certain column like for example:

`const records = await pb.collection('tickets').getFullList({
    sort: '-created',
});`

but i want to return the total list on a certain column in my tickets collection.
example of my column is "offices". I want to get all the total number of offices.

**Reply by ganigeorgiev** [reply]
It still doesn't make sense to me. What is `offices` in the first place? If it is a multiple `relation` why do you need a count and not just use the `record.offices.length` prop?

I'll stop engage further but if you need aggregations you'll have to create a helper View collection as shown above or a custom route, there is no other option at the moment.
